token-metadata/js/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@solana/spl-token-metadata",
     "description": "SPL Token Metadata Interface JS API",
-    "version": "0.1.0",
+    "version": "0.2.0",
```

Comment:
As I mentioned I'll bump this!

Reply:
Sure. Reverted back to 0.1.0 and i'll leave you to bump